src/components/charts/cardLineChart.tsx

```diff
@@ -75,7 +75,7 @@ const CardLineChart = ({ ratingData, title = '', displayLegend = false, height }
         <div className="bg-gray-800 relative flex flex-col min-w-0 break-words w-full shadow-lg rounded-lg px-4 py-3">
             <h2 className="text-white text-xl font-semibold text-center">{title}</h2>
             <div className={`mt-3 ${height}`}>
-                <canvas ref={chartContainer} className="h-full flex-auto" />
+                <canvas ref={chartContainer} className="h-full flex-auto width-full" />
```

Comment:
I think that `width-full` is not necessary here. I tried removing it and can't really see the change. What is the purpose of this?